.github/workflows/lint.yaml

```
@@ -3,48 +3,94 @@ on:
         with:
-          commit_message: "style: auto-fix by prek"
+          name: autofix-patch
+          path: |
+            autofix.patch
+            metadata.json
+          retention-days: 1
+      - name: Commit and push auto-fixes
+        if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
+        env:
+          PUSH_TOKEN: ${{ secrets.PAT }}
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)